src/test/lassipatanen/User/Profile.py

```diff
@@ -106,15 +128,16 @@ class ProfileResponse(CamelCaseModel):
         title="Nationality code",
         description="ISO 3166-1 alpha-2 code for nationality",
         example="FI",
+        nullable=True,
     )
     job_titles: Optional[List[str]] = Field(
```

Comment:
You probably don't want this to be `Optional` if you change the default to `...`, right?